fncs/antiND/safeStart.sqf

```diff
@@ -0,0 +1,22 @@
+/*
+Author: Pax_Jarome
+Locality:
+local execution
+local effect
+Desc:
+Checks if the receiver isn't a Headless Client, continues to set the players primary weapon to safe,
+copy magazine name, remove all bullets from the loaded magazine and adds a magazine of the previously
+loaded type to any free space on the player, thus preventing the player from discharging a bullet.
+This script will not accept parameters.
+This script does not return any values.
+*/
+
+waitUntil {not isNil player};
```

Comment:
You have to spawn this function because of this. `isNil` expression is definitly wrong